pkg/master/master.go

```diff
@@ -251,6 +253,7 @@ func (c completedConfig) New(delegationTarget genericapiserver.DelegationTarget)
 		batchrest.RESTStorageProvider{},
 		certificatesrest.RESTStorageProvider{},
 		extensionsrest.RESTStorageProvider{ResourceInterface: thirdparty.NewThirdPartyResourceServer(s, s.DiscoveryGroupManager, c.StorageFactory)},
+		networkingrest.RESTStorageProvider{},
```

Comment:
@danwinship if this came after the `extensions` group, `kubectl get networkpolicy` should return the `extensions/v1beta1` version of NetworkPolicy objects. Is that not the case?